packages/bpk-mixins/src/mixins/_badges.scss

```diff
@@ -105,3 +104,76 @@
     border-bottom-right-radius: 0;
   }
 }
+
+/// Success badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge();
+///     @include bpk-badge--success();
+///   }
+
+@mixin bpk-badge--success {
+  background-color: $bpk-color-green-500;
+}
+
+/// Destructive badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge();
+///     @include bpk-badge--destructive();
+///   }
+
+@mixin bpk-badge--destructive {
+  background-color: $bpk-color-red-500;
+  color: $bpk-color-white;
+}
+
+/// Light badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge();
+///     @include bpk-badge--light();
+///   }
+
+@mixin bpk-badge--light {
+  background-color: $bpk-color-gray-50;
+}
+
+/// Inverse badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge();
+///     @include bpk-badge--inverse();
+///   }
+
+@mixin bpk-badge--inverse {
+  background-color: $bpk-color-white;
+}
+
+/// Outline badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge();
+///     @include bpk-badge--outline();
+///   }
+
+@mixin bpk-badge--outline {
+  border: solid $bpk-border-size-sm $bpk-color-white;
+  background-color: rgba(0, 0, 0, .2);
```

Comment:
Can be addressed in a future PR but maybe we should have a token for absolute black, same as we do for white.

Reply:
Woops this should have been 20% opacity white, not 20% opacity black! (so `rgba(255, 255, 255, .2)`) 

And dear god no! We shouldn't use absolute black anywhere, and definitely shouldn't have a token for it! That'd be criminal!

Reply:
The reason I didn't use the white token is I'm not sure how to apply opacity to a token value. I had a quick google and it didn't seem to be something you can do without nesting divs

Reply:
Maybe we should have tokens for transparent versions of our colours, or add a function to `token-helper.js`?

Reply:
> The reason I didn't use the white token is I'm not sure how to apply opacity to a token value. I had a quick google and it didn't seem to be something you can do without nesting divs

You can do this natively in Sass:

`rgba(#fff, 0.2)`